action.yml

```diff
@@ -58,7 +67,7 @@ runs:
         echo "FAILURE_MESSAGE=$FAILURE_MESSAGE" >> $GITHUB_ENV
 
     - name: Report - success [on master]
-      if: job.status == 'success' && (github.ref == 'refs/heads/master' || inputs.always-notify == 'true')
+      if:  ${{ inputs.parent-job-status }} == 'success' && (github.ref == 'refs/heads/master' || inputs.always-notify == 'true')
```

Comment:
We should either refer to all inputs using `${{ ... }}` or without the brackets in order to be consistent